scripts/pi-hole/php/update_checker.php

```diff
@@ -59,7 +59,7 @@ function checkUpdate($currentVersion, $latestVersion)
     }
 
     // Get Pi-hole Docker Tag, if available
-    $docker_current = htmlspecialchars(getenv('PIHOLE_DOCKER_TAG'));
+    $docker_current = $versions['DOCKER_VERSION'];
```

Comment:
Not able to test currently, but just want to make sure there are no errors in the logs when we attempt to read a key that is not there?

Reply:
Your're a hound:

```
2022-09-16 16:01:41: (mod_fastcgi.c.421) FastCGI-stderr: PHP Notice:  Undefined index: DOCKER_VERSION in /var/www/html/admin/scripts/pi-hole/php/update_checker.php on line 62
```